handbook/communication/code_of_conduct.md

```diff
@@ -0,0 +1,188 @@
+# Code of Conduct
+
+[TOC]
```

Comment:
I don't think we need a TOC here, it will be automatically generated and shown on the right of the page based on headings.

Reply:
Gotcha, fixed